ord_schema/message_helpers_test.py

```diff
@@ -388,6 +388,113 @@ def test_identifier_getters(self):
                          _BENZENE_MOLBLOCK)
 
 
+class SetDativeBondsTest(parameterized.TestCase, absltest.TestCase):
+
+    def test_has_transition_metal(self):
+        xphos = reaction_pb2.Compound()
+        xphos.identifiers.add(type='SMILES',
```

Comment:
These look pretty good, but a couple of tips on writing tests:

* In general, try to make the test as simple as possible. It should be trivial for a reviewer to verify that the test is correct. For instance, instead of creating a `Compound` message with a complicated SMILES identifier, you could just do `self.assertFalse(message_helpers.has_transition_metal(Chem.MolFromSmiles('P')))`. AFAICT there's no reason to create the proto wrapper or use such a complicated structure for what you're testing here.
* Prefer parameterized tests over listing multiple test cases in the same method. See, for example, https://github.com/Open-Reaction-Database/ord-schema/blob/main/ord_schema/message_helpers_test.py#L95.

Reply:
>AFAICT there's no reason to create the proto wrapper or use such a complicated structure for what you're testing here.

I see, thanks. The only reason I added the wrapper was to make sure the method worked within the proto. I had already checked that the function worked in RDKit alone, but I see that's what we're going for in the test here too. Will update.

>Prefer parameterized tests over listing multiple test cases in the same method.

Sounds great. Will work on these now.

Reply:
Simplified/parameterized tests. Plan to add a couple more simple ones to `test_set_dative_bonds` tonight or tomorrow